hw/ip/prim/data/prim_mubi_pkg.sv.tpl

```diff
@@ -16,62 +16,62 @@ package prim_mubi_pkg;
 % for n in range(1, n_max_nibbles + 1):
 <%
   nbits = n * 4
-  hi_val = ''
-  lo_val = ''
+  true_val = ''
+  false_val = ''
   for k in range(1,n+1):
-    hi_val = ('5' if (k % 2) else 'A') + hi_val
-    lo_val = ('A' if (k % 2) else '5') + lo_val
+    true_val = ('A' if (k % 2) else '5') + true_val
+    false_val = ('5' if (k % 2) else 'A') + false_val
```

Comment:
@arnonsha this should align the values with the `lc_tx_t`.

Reply:
Great, thanks @msfschaffner 
LGTM